### ISSUE TITLE
Extract notification to own package

### ISSUE THREAD

**Issue by williamleven** [hulthe/google-ldap-sync#6]
Email notifications should be extracted to their own package

**Comment by williamleven** [comment]
Would probably be a good idea to use [Gotify](https://github.com/cthit/gotify)

**Comment by Portals** [comment]
Isn't Gotify in use not? @williamleven

**Comment by williamleven** [comment]
Nope: https://github.com/cthit/goldapps/blob/master/internal/pkg/services/admin/password.go
